TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/human/HumanActivity.java

```diff
@@ -1,13 +1,14 @@
-package com.novoda.tpbot.controller;
+package com.novoda.tpbot.human;
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 
 import com.novoda.notils.caster.Views;
+import com.novoda.tpbot.Direction;
 import com.novoda.tpbot.R;
 import com.novoda.tpbot.SelfDestructingMessageView;
 
-public class BotControllerActivity extends AppCompatActivity {
+public class HumanActivity extends AppCompatActivity {
```

Comment:
I'm not entirely sure about this naming:
I kinda liked the `controller - receiver` idea, but on the other hand I like your update `human - machine`